cmake/helpers.cmake

```
@@ -63,6 +63,12 @@ if(CMAKE_SYSTEM_NAME STREQUAL Linux)
     check_cxx_compiler_flag("-fstack-clash-protection" CXX_HAS_FSTACK_CLASH_PROTECTION)
 endif()
 
+if (UR_USE_CFI AND UR_USE_ASAN)
+    message(WARNING "Both UR_USE_CFI and UR_USE_ASAN are ON."
```

cmake doesn't insert a space or line break between the strings
```suggestion
    message(WARNING "Both UR_USE_CFI and UR_USE_ASAN are ON. "
```